src/nwjs-autoupdater/main.go

```diff
@@ -11,14 +11,13 @@ import (
 )
 
 func main() {
-	var bundle, instDir string
+	var bundle, instDir, appName string
 
 	flag.StringVar(&bundle, "bundle", "", "Path to the update package")
 	flag.StringVar(&instDir, "inst-dir", "", "Path to the application install dir")
+	flag.StringVar(&appName, "app-name", "", "Application executable name")
```

Comment:
@rsfernandes Can you please use `my_app` as a default value here? This will not break autoupdater for people who use it without --app-name flag :)